pydatalab/src/pydatalab/apps/ftir/__init__.py

```diff
@@ -0,0 +1,77 @@
+import os
+from pathlib import Path
+
+import bokeh.embed
+import pandas as pd
+import numpy as np
+from bokeh.models import HoverTool, LogColorMapper
+
+from pydatalab.blocks.base import DataBlock
+from pydatalab.bokeh_plots import DATALAB_BOKEH_THEME, selectable_axes_plot
+from pydatalab.file_utils import get_file_info_by_id
+from pydatalab.logger import LOGGER
+
+
+def parse_ftir_asp(filename: Path):
```

Comment:
Could you just add a little docstring here that describes the assumptions of the format?

```suggestion
def parse_ftir_asp(filename: Path) -> pd.DataFrame:
    """Parses .asp FTIR data from e.g., Agilent spectrometers.
    
    The file consists of header numbers followed by a 1D array of absorbance data
    ...
    ...
    etc.
    
    Returns:
       A dataframe with columns for Wavenumber and Absorbance in units x, y, z
```